packages/@okta/vuepress-site/docs/reference/api/groups/index.md

```diff
@@ -849,10 +849,9 @@ curl -v -X PUT \
 
 <ApiOperation method="delete" url="/api/v1/groups/${groupId}" />
 
-Removes a group with `OKTA_GROUP` type from your organization.
+Removes a group with `OKTA_GROUP` or `APP_GROUP` type from your organization.
```

Comment:
There are no other possible group types except those two, right?  There's a chance this gives the impression that there are other types, that cannot be removed. Maybe:
"Removes a group from your organization . Groups of both types, `OKTA_GROUP` and `APP_GROUP`, can be removed. "

Reply:
We don't have more group types today, but we might in the future. So may be worth keeping this specific to Okta and App groups.